code/controllers/subsystem/dbcore.dm

```diff
@@ -77,7 +77,7 @@ SUBSYSTEM_DEF(dbcore)
 	while(length(processing_queries))
 		var/datum/db_query/query = popleft(processing_queries)
```

Comment:
make this store weakrefs and just check the query datum still exists, this only happens because of queries running on clients and honestly unless we want to ban running queries from the scope of clients they will always have this issue.

or we could define a Del() proc on queries and have the code properly detect when they gc from lack of references, avoiding the need for this entire system and check.